.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: node_js
 node_js:
-      - "0.12"
-      - "4"
-      - "5"
-      - "6"
-      - "7"
+      - "10"
+      - "12"
+      - "14"
+      - "15"
```

Comment:
The node versions here were ancient. I updated them because the new test tooling just simply won't run in old unsupported versions.